kernel/obj_core.c

```diff
@@ -314,10 +314,10 @@ int k_obj_stats_disable(struct k_obj_core *obj_core)
 	return rv;
 }
 
-int k_obj_stats_enable(struct k_obj_core *obj_core)
+int k_obj_core_stats_enable(struct k_obj_core *obj_core)
 {
 	int  rv;
-	struct k_obj_stats_desc *desc;
+	struct k_obj_core_stats_desc *desc;
```

Comment:
Oh, and here they are fixed. Maybe just squash the changes in this file to the previous patch?

Reply:
Will do.